GameEngine/Engine.cs

```diff
@@ -43,9 +43,27 @@ public static void Start(Func<IGameData> gameDataFactory, int loadFromSlot = 0)
                 {
                     // Otherwise if we're not starting a new game, show the game intro text
                     // Show the game introduction
+                    string finishedChange = gameData.FinalName();
+                    string answer;
                     string intro = gameData.GetGameIntroduction();
+                    string playerData = gameData.MainCharacterNamePick();
                     Console.Clear();
                     Console.WriteLine(intro);
+                    Console.WriteLine(playerData);
+                    answer = Console.ReadLine();
+                    answer = answer.ToLower();
```

Comment:
Can you use StringComparison.OrdinalIgnoreCase here instead?

Reply:
ok